docs/apm/traces/tracing-ingest.md

```diff
@@ -0,0 +1,59 @@
+---
+id: tracing-ingest
+title: Use Tracing to Troubleshoot Ingest
+sidebar_label: Use Tracing to Troubleshoot Ingest
+description: Use tracing to to search the Audit Event Index for all events that impact ingest.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The [Audit Event Index](/docs/manage/security/audit-event-index/) contains event logs in JSON format on account activities, allowing you to monitor and audit changes. You can use the `tracingIngest` source category to search the Audit Event Index for all events that impact [log ingestion](/docs/manage/ingestion-volume/log-ingestion/).
```

Comment:
These events don't impact log ingestion. In Audit Event Index we store logs about events, but events described in this page are correlated to the tracing ingest.

Reply:
I don't think we have a page for trace ingestion like this log one (at least I didn't find it), but maybe we can link here to the [Setup and Configuration, point 1](https://help.sumologic.com/docs/apm/traces/get-started-transaction-tracing/#step-1-set-up-traces-collection) as it appears to have a similar role.

Reply:
Done.